requirements-dev.txt

```diff
@@ -18,7 +18,6 @@ cryptography==43.0.1
 dill==0.3.9
 distro==1.9.0
 docker==7.1.0
-flake8==7.1.1
```

Comment:
Those are `flake8` install_requirements, are other tools still rely on them if you delete it ?
```
    mccabe>=0.7.0,<0.8.0
    pycodestyle>=2.12.0,<2.13.0
    pyflakes>=3.2.0,<3.3.0
```

Reply:
Not entirely sure, but once I merge in some uv-based changes those will be automatically recalculated.